src/helpers/analytics.coffee

```diff
@@ -0,0 +1,64 @@
+_ = require 'underscore'
+Router = require 'react-router'
+{HistoryLocation} = require 'react-router'
+DestinationHelper = require '../helpers/routes-and-destinations'
+
+# generate custom event data for routes
+Events =
+  viewTaskStep: ({courseId}) ->
+    # track all work done on a course
+    Analytics.sendEvent 'Course', 'Work', label: courseId
+
+  viewStudentDashboard: ({courseId}) ->
+    # compare activity between courses
+    Analytics.sendEvent 'Student', 'Dashboard', label: courseId
+
+
+# Translators convert a url like '/foo/bar/123/baz/1' into a simplified one like just '/foo/bar'
+Translators =
```

Comment:
Sorry, my comment was a little muddled.. The `<Route>` component has 2 fields on it that are already "unique-ish", the  path and name. I was wondering if we could use one of those instead instead of creating another list here.